feature/community/build.gradle

```diff
@@ -28,8 +30,24 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.2.0'
+    }
 }
 
 dependencies {
+    implementation libs.bundles.compose
+    implementation libs.compose.navigation
```

Comment:
You don't have to import this library like this, because you already have the whole compose bundle, where you already have it

Reply:
yes It was as a test, I already removed from, I understand compose and navigation is already bundled inside the lib.bundles.compose variable. thanks